JavaScript DOM/Bom task/born.js

```diff
@@ -0,0 +1,36 @@
+function printWindowSize() {
+  console.log(window.innerHeight, window.innerWidth);
+}
+
+function printEnvInfo() {
+  for (let propName in window.navigator) {
+    let propValue = window.navigator[propName];
+    if (propValue?.constructor === String) {
+      console.log(propName, propValue);
+    }
+  }
+}
+
+function getSearch(href) {
+  let signIndex = href.indexOf("?") === -1 ? href.indexOf("#") : href.indexOf("?");
```

Comment:
Please use 'const' as default and 'let' only if necessary. Please review and fix this issue in other places